### PR TITLE
Use Local for WIN32 FileCache

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: a737679326453b7b96e8b7a12b5087479a4d7bf0 available at: https://gist.github.com/f7938f5ed194c7673d0c4c13f3f6dbf0<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9868-a737679.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9868-a737679.dmg

**Comment by AndrewMeadows** [comment]
OSX build failed for this link error:
```
Undefined symbols for architecture x86_64:
  "cache::FileCache::MAX_UNUSED_MAX_SIZE", referenced from:
      cache::FileCache::setUnusedFileCacheSize(unsigned long) in libnetworking.a(FileCache.o)
      cache::FileCache::setOfflineFileCacheSize(unsigned long) in libnetworking.a(FileCache.o)
ld: symbol(s) not found for architecture x86_64
```

File: libraries/shared/src/PathUtils.cpp [hunk 1]
```diff
@@ -30,11 +30,15 @@ const QString& PathUtils::resourcesPath() {
...
 }
 
-QString PathUtils::getRootDataDirectory() {
+QString PathUtils::getRootDataDirectory(bool roaming) {
     auto dataPath = QStandardPaths::writableLocation(QStandardPaths::HomeLocation);
 
 #ifdef Q_OS_WIN
-    dataPath += "/AppData/Roaming/";
+    if (roaming) {
+        dataPath += "/AppData/Roaming/";
```
Comment (jherico) [inline libraries/shared/src/PathUtils.cpp]: hardcoding these paths is a bad idea (even though they were hardcoded before).

`QStandardPaths::AppLocalDataLocation` should provide the _local_ location, while `QStandardPaths::AppDataLocation` should provide the _roaming_ path.

**Comment by hifi-gustavo** [comment]
Build console output for commit: af2c409e90d247df2a666886fb03aea8e5e4fadb available at: https://gist.github.com/40cf1e31a55c63e6b8dcc363da84f5f8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9868-af2c409.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9868-af2c409.dmg

**Comment by zzmp** [comment]
Tested on WIN32 - proper paths are used for Roaming and Local.

**Comment by zzmp** [comment]
I am unable to test on OSX (don't have a machine handy) - if you have that, please include it in CR.

**Comment by zzmp** [comment]
I'm testing on Ubuntu right now as well.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 808973d7d25087341e9058f509ecfe8544243f68 available at: https://gist.github.com/40cf1e31a55c63e6b8dcc363da84f5f8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9868-808973d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9868-808973d.dmg

**Pull request merged**